tests/config_test.go

```diff
@@ -112,7 +112,12 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				expectedOutput := "value1value2value3"
 
 				By("Running VMI")
-				vmi := libvmifact.NewAlpine(libvmi.WithConfigMapDisk(configMapName, configMapName))
+				vmi := libvmifact.NewAlpineWithTestTooling(libvmi.WithConfigMapDisk(configMapName, configMapName))
+				for i := range vmi.Spec.Domain.Devices.Disks {
+					if strings.HasPrefix(vmi.Spec.Domain.Devices.Disks[i].Name, "configmap-") {
+						vmi.Spec.Domain.Devices.Disks[i].Serial = "configmap-disk"
+					}
```

Comment:
The test inputs aren't immediately clear. IMO, it would improve clarity to define them explicitly.
Given that `libvmi.WithConfigMapDisk` currently lacks the ability to set the disk's serial, please consider the following alternative:

```suggestion
disk := ...
volume := ...
vmi := libvmifact.NewAlpineWithTestTooling(libvmi.WithDisk(disk), libvmi.WithVolume(volume))
```

Same bellow.

Reply:
Done